src/abi.rs

```diff
@@ -138,6 +138,13 @@ impl<'gcc, 'tcx> FnAbiGccExt<'gcc, 'tcx> for FnAbi<'tcx, Ty<'tcx>> {
             if attrs.regular.contains(rustc_target::abi::call::ArgAttribute::NonNull) {
                 non_null_args.push(arg_index as i32 + 1);
             }
+            if let Some(align) = attrs.pointee_align {
```

Comment:
I don't believe this is correct to put this here since it will be ignored when optimizations are disabled.